Helm: improve implementation and documentation of  helm.sh (#15957)
rook/rook: Documentation/Contributing/development-environment.md

## Install Helm

Use [helm.sh](https://github.com/rook/rook/blob/master/tests/scripts/helm.sh) to install Helm

> - To install Helm run `tests/scripts/helm.sh up`.

Author: @travisn  wrote:

> Since this installs to a tmp folder, not sure we want to recommend this for development. What if we just give a link to the helm downloads so they can go install it separately? Then we can simplify this whole section of the dev guide.

done. I tried to simplify it.

> And let's see if we can separately remove the need for helm.sh from the CI.

Maybe we can try that once this change here is agreed upon.
I am however afraid that it might be problem because the temporary folder the script install helm into is referenced in som of the code under `tests/`

## Install Helm

> You will  need the `helm` tool to use the helm charts rook provides. Depending on your  operating system,

Reviewer: ```suggestion
Install the `helm` tool to use the helm charts rook provides. Depending on your operating system,
```

> install it from system pckages or follow the [official guide](https://helm.sh/docs/intro/install/)

Reviewer: ```suggestion
install it from system packages or follow the [official Helm guide](https://helm.sh/docs/intro/install/).
```
